### PR TITLE
Add /siblings endpoint

### LINKED ISSUE #109: Siblings endpoint
To implement partial tree synchronisation in client library, it is required to have a `/siblings?index=${index}` endpoint in relayer which returns all sibling internal tree nodes for a provided leaf index.

### PR DESCRIPTION
**Pull request by LyzeOfKiel**
Changes:
* Add `/siblings` endpoint. It returns hex-encoded left sibling nodes starting from the specified leaf index up to the tree root. Requested index `i` should pass following checks: `i > 0`, `i % 128 = 0`, `i < tree.next_index`, otherwise `400 Bad Request` is returned.
Closes #109 

Depends on https://github.com/zkBob/libzkbob-rs/pull/33

### CODE REVIEW

File: zp-relayer/router.ts [hunk 1]
```diff
@@ -37,6 +38,7 @@ router.get('/job/:id', wrapErr(endpoints.getJob))
 router.get('/info', wrapErr(endpoints.relayerInfo))
 router.get('/fee', wrapErr(endpoints.getFee))
 router.get('/limits', wrapErr(endpoints.getLimits))
+router.get('/siblings', wrapErr(endpoints.getSiblings))
```
Comment (akolotov) [inline zp-relayer/router.ts:41]: makes sense to reflect the new route in the README file

Comment (LyzeOfKiel) [reply zp-relayer/router.ts:41]: Updated

**Review by akolotov — APPROVED** [review]

File: zp-relayer/endpoints.ts [hunk 2]
```diff
@@ -241,6 +242,25 @@ async function getLimits(req: Request, res: Response) {
...
+  const errors = checkGetSiblings(req.query)
+  if (errors) {
+    logger.info('Request errors: %o', errors)
+    res.status(400).json({ errors })
+    return
+  }
+
+  const index = req.query.index as unknown as number
+
+  if (index >= pool.state.getNextIndex()) {
```
Comment (k1rill-fedoseev) [inline zp-relayer/endpoints.ts:255]: Is it possible that newly registering user will observe a new tree root faster than the relayer? This case will trigger an error on the relayer and the user fast sync will not work, right?

Comment (LyzeOfKiel) [reply zp-relayer/endpoints.ts:255]: >Is it possible that newly registering user will observe a new tree root faster than the relayer? 

As client primarily communicates with the relayer (and not the contract directly), and they fetch the root before requesting siblings, this should not be possible. Am I missing something?

Anyways, if we try to request siblings removing this check and `index` will be `>= next_index` we will get an `libzkbob-rs` error `Tree is undefined at index ${index}`

Comment (k1rill-fedoseev) [reply zp-relayer/endpoints.ts:255]: > they fetch the root before requesting siblings, this should not be possible

If that's the case, then we are good. Just wanted to make sure that client will not get any weird errors.

**Review by r0wdy1 — APPROVED** [review]

**Pull request merged**